core/math/math_defs.h

```diff
@@ -56,7 +56,7 @@
 #define UNIT_EPSILON 0.001
 #endif
 
-#define USEC_TO_SEC(m_usec) ((m_usec) / 1000000.0)
+#define USEC_TO_SEC(m_usec) ((m_usec) * 0.000'000'1)
```

Comment:
Should be 0.000'001

Reply:
Good catch :slightly_smiling_face: Fixed. Unfortunately, the reported FPS value is still wrong.